src/Data/Event/EventDataClass.ts

```diff
@@ -0,0 +1,47 @@
+import { provideLocalStorageDataClass } from '../../utils/provideLocalStorageDataClass'
+import spiderImage from './FakeBinaries/Schulung_Spider_30.png'
+import roadshowImage from './FakeBinaries/TYNACOON_Roadshow.jpg'
+
+export const Event = provideLocalStorageDataClass('Event', {
+  initialContent: [
+    {
+      _id: 'DD6585A543FE48018F497D3631332C6F',
+      keyword: 'Schulung: Wirkungsprinzipien Typ Spider 30',
+      number: 'P-11-007950',
+      beginsAt: '2024-04-29T13:30:00Z',
+      endsAt: '2024-04-29T14:30:00Z',
+      location: 'Schulungszentrum 1',
+      organizer: '',
+      responsibleAgent: '0BD5DC0F9DA442C5946FA7ECAF870D7B',
+      status: 'PSA_PRO_EVT_REG_OPN',
+      statusLocalized: 'registration open',
+      url: '',
+      logo: { url: spiderImage },
```

Comment:
* [ ] FYI this is a good example of how images from external backends can easily trash an otherwise well-designed portal. I guess giving editors some control over how external images are displayed is something we should add at some point.

Reply:
We do already have the plan to provide more information for an event from the CMS content. But I'm not sure if this also includes the image or not.

Reply:
Since CMS content can include images - sounds good!